src/platform/utilities/feature-toggles/index.jsx

```diff
@@ -15,9 +15,7 @@ function makeEnvironmentToggleValues(env = environments) {
 }
 
 function getBootstrappedToggleValues() {
-  return {
-    facilityLocatorShowCommunityCares: true,
-  };
+  return window.featureToggles ? window.featureToggles : {};
```

Comment:
the `featureToggles` are bootstrapped into the static html. The featureToggles will use the bootstrapped values for the initial page render. 

Todo here is to add a helper to grab the loading state of the feature toggles. If we're toggling something that appears on the initial page render, then the UX will need to account for that and wait for the `FlipperClient` to fetch the values in the browser rather than using the bootstrapped values added during the build process- i.e. it will need to render a loading state until the latest values are fetched.